src/js/hca/config/form.js

```diff
@@ -274,8 +278,8 @@ const formConfig = {
             properties: {
               gender: {
                 type: 'string',
-                'enum': genders.map(gender => gender.value),
-                enumNames: genders.map(gender => gender.label)
+                'enum': Object.keys(genderLabels),
```

Comment:
If we import `gender` from fullSchemaHca.properties along with the other fields, we could change the code to look like:

```js
properties: {
  gender,
  ...
```
I think that would simply things a bit.